src/test/java/core/basesyntax/strategy/handlers/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.strategy.handlers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.db.Storage;
+import org.junit.After;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    private SupplyOperationHandler handler = new SupplyOperationHandler();
```

Comment:
In some tests, you initialize objects using a static initialization block, in others through the @Before method, and here immediately upon creation. I suggest doing the same everywhere. My advice is using the @Before method